### ISSUE TITLE
default quality

### ISSUE THREAD

**Issue by dmsnova** [mkvd/smarttv-twitch#66]
Is there an easy way to change it in the code for the default quality to be something specific not just source to be default at the first launch of the programe

**Comment by henriquelbsouza** [comment]
Here the default quality is the last one I selected.
If I used the program and selected Medium, the next time I open the default will be Mediu... Unless the channel only have source (new channels), but that is it...

Or you want to do that like the first of all times that you open the twitch? Really don't see the point, but probably you can do that...

**Comment by dmsnova** [comment]
Yeah, it's after i power on the TV and launch the program. Unfortunately the internet connection in my place right here can't do 'source' quality, so i have to manually change it every time I power on the tv. I looked through the files but didn't find how to do it.

**Comment by henriquelbsouza** [comment]
But that is the way it works right now... If you change to Low quality that will be the default quality next time, at least in my TV it is working like this...

You know press the "<" button and then channel arrow down... I will take a look if I did any other thing, but I don't think so...
